lib/derivations.nix

```diff
@@ -61,26 +61,37 @@ in
         (lazyDerivation { inherit derivation }).passthru
 
         (lazyDerivation { inherit derivation }).pythonPath
+    # Inputs
 
+    structured function argument
+
+    : derivation
+      : The derivation to be wrapped.
+
+    : meta
+      : Optional meta attribute.
+
+        While this function is primarily about derivations, it can improve
+        the `meta` package attribute, which is usually specified through
+        `mkDerivation`.
+
+    : passthru
+      : Optional list of assumed outputs. Default: ["out"]
+
+        This must match the set of outputs that the returned derivation has.
+        You must use this when the derivation has multiple outputs.
```

Comment:
```suggestion
      : Optional extra values to add to the returned attrset.

        This can be used for adding package attributes, such as `tests`.
```